lib/menu.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Menu do
+  def game_types do
+    display_game_types
+
+    get_value_betweenf(0, Enum.count(games) - 1)
```

Comment:
`betweenf` ?
And I would probably use a variable to explain `Enum.count(games) - 1`